plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/CloudSdkServer.java

```diff
@@ -70,8 +70,7 @@
   public static CloudSdkServer getCloudSdkServer(IServer server) {
     CloudSdkServer cloudSdkServer = server.getAdapter(CloudSdkServer.class);
     if (cloudSdkServer == null) {
-      cloudSdkServer = (CloudSdkServer) server.loadAdapter(CloudSdkServer.class,
-                                                           new NullProgressMonitor());
+      cloudSdkServer = (CloudSdkServer) server.loadAdapter(CloudSdkServer.class, null);
```

Comment:
Is it possible that the `loadAdapter` method returns `null`? If so, we should guard against it in the caller methods.